### PR TITLE
fix(Card): updated v5 logic to prevent unclickable cards

### PR DESCRIPTION
**Pull request by thatblindgeye**
**What**: Towards #9875 

**Additional issues**:

### CODE REVIEW

**Review by kmcfaul — APPROVED** [review]

File: packages/react-core/src/components/Card/Card.tsx [hunk 1]
```diff
@@ -14,15 +14,21 @@ export interface CardProps extends React.HTMLProps<HTMLElement>, OUIAProps {
...
-  /** Modifies the card to include selected styling */
-  isSelected?: boolean;
-  /** Modifies the card to include clickable styling */
+  /** Flag indicating that the card is clickable and contains some action that triggers on click. */
   isClickable?: boolean;
-  /** Modifies a clickable or selectable card to have disabled styling. */
+  /** Flag indicating whether a card that is both clickable and selectable is currently selected and has selected styling.
+   * This will not determine the card's actual selected state.
+   */
+  isSelected?: boolean;
```
Comment (tlabaj) [inline packages/react-core/src/components/Card/Card.tsx:26]: We may want to deprecate this prop and come up with a better name.  The prop not determining the selected state is confusing given the name.

File: packages/react-core/src/components/Card/examples/Card.md [hunk 2]
```diff
@@ -109,37 +109,39 @@ A common use case of this is to set all but one body section to `isFilled={false
...
 
 ```ts file='./CardSingleSelectable.tsx'
 
 ```
 
-### Clickable
+### Actionable
 
-A card can perform an action or navigate to a link by clicking anywhere within the card. You can also pass in the `isExternalLink` property to `selectableActions` if you want a clickable card's link to open in a new tab or window.
+An actionable card can perform an action or navigate to a link by clicking anywhere within the card. You can also pass in the `isExternalLink` property to `selectableActions` if you want an actionable card's link to open in a new tab or window.
```
Comment (tlabaj) [inline packages/react-core/src/components/Card/examples/Card.md:130]: @edonehoo do you mind taking a look at this copy.

File: packages/react-core/src/components/Card/Card.tsx (hunk 1 above)
Comment (thatblindgeye) [reply packages/react-core/src/components/Card/Card.tsx:26]: True. We could keep this as is and I can update the logic so that isSelected determines the `isChecked` value within CardHeader (essentially what I'm doing with the isClicked prop for "clickable" cards). That would just require deprecating the `isChecked` prop in CardHeader's `CardHeaderSelectableActionsObject` interface. Doing that would be a little more consistent with the API with this update; rather than needing to pass a prop in CardHeader tor selectable cards and a prop on Card for clickable cards. WDYT?

Comment (tlabaj) [reply packages/react-core/src/components/Card/Card.tsx:26]: Yes, that sounds like it would be more intuitive for the consumer.

File: packages/react-core/src/components/Card/examples/Card.md (hunk 2 above)
Comment (edonehoo) [reply packages/react-core/src/components/Card/examples/Card.md:130]: I like it! I might further reword it to be a little more user action oriented like:

"An actionable card can perform an action or navigate to a link when a card is clicked. To open a link in a new tab or window, pass the `isExternalLink` property to `selectableActions`. "

Comment (thatblindgeye) [reply packages/react-core/src/components/Card/examples/Card.md]: Update the verbiage, though I kept in the "by clicking anywhere within the card" bit since the I mention it for the selectable card, and it's one of the differentiating aspects of the "selectable" only/"actionable" only cards and the "selectable and actionable" cards. Can also just remove that verbiage from both example descriptions intead, though.

Comment (edonehoo) [reply packages/react-core/src/components/Card/examples/Card.md]: ah, I see what you mean. It also makes to me to leave your original phrasing in that case!

**Review by kmcfaul — APPROVED** [review]

**Review by tlabaj — APPROVED** [review]
